upstairs/src/lib.rs

```diff
@@ -132,6 +174,22 @@ pub trait BlockIO: Sync {
     }
 }
 
+/// Await on multiple BlockReqWaiters in parallel
+#[inline]
+pub async fn wait_all(
```

Comment:
By using the `wait_all`, does it mean that the work that was put into the `iter` won't start until the `wait_all` is called?

Reply:
In this case, this is waiting on the operations completing, not starting.